.travis.yml

```diff
@@ -188,6 +184,14 @@ install:
 script:
   - travis_wait 45 suite/runsuite_wrapper.pl travis $EXTRA_ARGS
 
+after_script:
+  - sleep 10
+  - echo "--end--"
+
+after_failure:
+  - sleep 10
+  - echo "--end--"
```

Comment:
Could these be made specific to the Mac job only?

Reply:
Does it not work to put this inside the job matrix and so local to the Mac job w/o needing a global var to connect them?

Reply:
Here, `sleep` keeps the VM around long enough for all logs to be captured. As build is done after the job matrix is set up, there aren't any pending logs at that point, so IIUC moving `sleep` to the job matrix won't work.

Reply:
Making sure we're talking about the same thing: I mean putting an `after_script` and `after_failure` config setting inside the Mac job entry, not some other way of adding a sleep.  You're saying Travis won't accept `after_script` inside the matrix for a job-specific config?

Reply:
Sorry, I misunderstood your suggestion. Made the changes. Thanks.

Reply:
NP.  Thanks for fixing: IMHO it's cleaner this way.